tests/13-ieee802154/js/security.js

```diff
@@ -0,0 +1,26 @@
+TIMEOUT(50000);
+
+var failed = false;
```

Comment:
Calling `log.testFailed()` will terminate the test, so this variable shouldn't be required now.